web/app/(commonLayout)/app/(appDetailLayout)/[appId]/overview/cardView.tsx

```diff
@@ -33,18 +33,21 @@ const CardView: FC<ICardViewProps> = ({ appId }) => {
   if (!response)
     return <Loading />
 
-  const handleError = (err: Error | null) => {
+  const handleError = (err: Error | null, msgs = {
+    success: 'modifiedSuccessfully',
+    fail: 'modificationFailed',
```

Comment:
`success` and `filed` should not appear at the same time,  so I think defining this function like the suggestion below would make more sense.
```typescript
const handleError = (param: 
    | { success: 'modifiedSuccessfully',  err: null }
    | { failed: 'modificationFailed', err: Error
  }) => { *** }
```

Reply:
> `success` and `filed` should not appear at the same time, so I think defining this function like the suggestion below would make more sense.
> 
> ```ts
> const handleError = (param: 
>     | { success: 'modifiedSuccessfully',  err: null }
>     | { failed: 'modificationFailed', err: Error
>   }) => { *** }
> ```

@MatriQ  you are right ！！